python/pacemaker/_cts/CTS.py

```diff
@@ -2,7 +2,9 @@
 
 __all__ = ["CtsLab", "NodeStatus", "Process"]
 __copyright__ = "Copyright 2000-2023 the Pacemaker project contributors"
-__license__ = "GNU General Public License version 2 or later (GPLv2+) WITHOUT ANY WARRANTY"
+__license__ = (
+    "GNU General Public License version 2 or later (GPLv2+) WITHOUT ANY WARRANTY"
+)
```

Comment:
Yuck. I'm fine with extending our line length to 100. I think 80 is a bit short for Python since it's so line-oriented.

Reply:
black already wraps at about 88.  Bumping to 100 fixes a lot of the little things (like `__license__`), but the big ones mostly still remain.  That's improvement, though.